static/app/views/performance/newTraceDetails/traceTypeWarnings/errorsOnlyWarnings.tsx

```diff
@@ -191,7 +193,9 @@ function PerformanceQuotaExceededWarning(props: ErrorOnlyWarningsProps) {
 
   const title = tct("You've exceeded your [billingType]", {
     billingType: subscription?.onDemandBudgets?.enabled
-      ? t('pay-as-you-go budget')
+      ? subscription.planTier === 'am3'
```

Comment:
It might be better to match `on-demand budget` to AM2 and AM1 tiers, as we're using the pay-as-you-go budget branding from AM3 onwards.

Reply:
good point!!